### PR TITLE
Add passenger menu group

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/menuGroups/PassengerMenuGroup.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export default class PassengerMenuGroup extends React.Component<Props> {
+  render = () => (
+    <TitledMenuGroup title={<Translation id="mmb.main_menu.passengers" />}>
+      <MenuItem
+        onPress={VoidAction} // TODO: Add handler
+        title={<Translation id="mmb.main_menu.details" />}
+        isActive={false} // TODO: Add logic
+        icon={<TextIcon code="&#xe087;" />}
+      />
```
Comment (mrtnzlml) [inline app/MMB/src/menuGroups/PassengerMenuGroup.js]: TODO: bags (? not sure about that), passengers details + visa info

screen shot 2018-05-15 at 15 15 46

Comment (tbergq) [reply app/MMB/src/menuGroups/PassengerMenuGroup.js]: Maybe bags shows up there if it is ordered?

Comment (mrtnzlml) [reply app/MMB/src/menuGroups/PassengerMenuGroup.js]: That sounds reasonable but you can find the bags in the passenger as well. Not sure about the flow here... 🤷‍♂️

Comment (tbergq) [reply app/MMB/src/menuGroups/PassengerMenuGroup.js]: Maybe some bags contains passengers? 🙈

**Pull request merged**